python/sglang/srt/layers/moe/topk.py

```diff
@@ -122,9 +124,20 @@ def grouped_topk(
     )  # [n, e]
     tmp_scores = scores.masked_fill(~score_mask.bool(), 0.0)  # [n, e]
     topk_weights, topk_ids = torch.topk(tmp_scores, k=topk, dim=-1, sorted=False)
+    if share_fusion:
+        topk_ids[:, -1] = torch.randint(low=num_experts,
+                                        high=num_experts + share_fusion,
+                                        size=(topk_ids.size(0), ),
+                                        dtype=topk_ids.dtype,
+                                        device=topk_ids.device)
+        topk_weights[:, -1] = topk_weights[:, :-1].sum(dim=-1) * 1.0 / 2.5
```

Comment:
nit: curious what is `1.0/2.5`, would be great to add a line of comment explaining it

Reply:
2.5 is the `routed_scaling_factor`(https://huggingface.co/deepseek-ai/DeepSeek-R1/blob/main/config.json#L57), I'll refine the code later.